projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -329,11 +326,6 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
   updateRequired(): void {
     this.subs.push(
       this.workflowForm.valueChanges.subscribe(() => {
-        let fieldsArray: Array<WorkflowFormFieldComponent> | undefined = undefined;
-        if (this.inputFields) {
-          fieldsArray = this.inputFields.toArray();
-        }
-        this.workflowStepAllowed(fieldsArray);
```

Comment:
Shouldn't this be replaced by a call to hasGapInSequence? So that it has the required side-effect?

Reply:
It's no longer needed, and in any case it was firing after validation had already occurred (which is why the gap in sequence error showed only after TWO gaps were created)